asset-transfer-basic/application-typescript/package.json

```diff
@@ -20,11 +20,13 @@
     "author": "Hyperledger",
     "license": "Apache-2.0",
     "dependencies": {
-        "fabric-ca-client": "^2.2.4",
-        "fabric-network": "^2.2.4"
+        "fabric-gateway": "0.1.0-dev.20210921.3"
```

Comment:
Can we use a latest or unstable instead of pinning to a point in time version?

Reply:
Ideally we would use **latest** now that (as of yesterday) **latest** does actually point to the most recent build rather than one from 6 months ago! However... until we reach a first release version, this may cause breakages if any API changes are made in the development stream, so we might be safer picking a specific build version for now, _as long as_we remember to update again (and change to use **latest** tag) when we get to release time._